crates/wiggle/src/lib.rs

```diff
@@ -15,6 +15,9 @@ mod borrow;
 mod error;
 mod guest_type;
 mod region;
+pub mod tracing {
```

Comment:
I think this could perhaps be `pub extern crate tracing;`?

Reply:
Thanks, didn't know that was the preferred syntax but it is a lot clearer. Fixed.